practice_05.25.19_interface/Task2_BankAccount/BankAccount.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace practice_05._25._19_interface.Task2_BankAccount
+{
+    public class BankAccount
+    {
+        protected decimal currentBalance;
```

Comment:
Please try to leave all fields as private.
I'd suggest to make protected property.

Reply:
fixed